### PR TITLE
Cleanup ModuleTemplateLoader

### PR DESCRIPTION
**Pull request by sarjon**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Code clean up
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | n/a
| How to test?  | It should still load templates from modules.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9282)

### CODE REVIEW

**Comment by sarjon** [comment]
ping @mickaelandrieu , it needs your review. :)

File: src/PrestaShopBundle/Twig/Locator/ModuleTemplateLoader.php [hunk 1]
```diff
@@ -34,38 +34,28 @@
...
      */
-    private $rootPath;
-
-    /**
-     * @param array  $namespaces A collection of path namespaces with namespace names.
-     * @param string|array $paths    A path or an array of paths where to look for templates
-     * @param string|null  $rootPath The root path common to all relative paths (null for getcwd())
-     */
-    public function __construct(array $namespaces, $paths = array(), $rootPath = null)
+    public function __construct(array $namespaces, array $modulePaths = array())
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Twig/Locator/ModuleTemplateLoader.php:40]: don't you forget the yaml declaration? :)

Comment (sarjon) [reply src/PrestaShopBundle/Twig/Locator/ModuleTemplateLoader.php:40]: hmm, maybe i do. :/ it seemed that this code was not used, am i  missing something?

Comment (mickaelandrieu) [reply src/PrestaShopBundle/Twig/Locator/ModuleTemplateLoader.php:40]: after verification, no => https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Resources/config/services/bundle/twig.yml#L7

**Review by mickaelandrieu — DISMISSED** [review]

**Review by mickaelandrieu — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
> It should still load templates from modules.

It's unit tested, this doesn't need any test from QA team.

**Review by mickaelandrieu — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
Thanks @sarjon !

**Pull request merged**